File: C/12.09.24/palindrome.c
 //wap to check a no. is palidrome or not
#include<stdio.h>
void main()
{
     int n,rem,x,s=0;
     printf("enter the no=");
     scanf("%d",&n);
     for(x=n ;n>0;n=n/10)
     { 
       rem=n%10;
       s=(s*10)+rem;
       n=x;
     }
      if(n==s)
      {
       printf("%d is palindrome number");       
      }   
      else
      {
       printf("%d is not palindrome number");       
      }
      
}
     

File: JAVA/Basic/palindrome.java
import java.util.Scanner;

public class palindrome{
    
    public static void main(String[] as)
    {
        int n,x, rem=0,s=0;
        Scanner ob = new Scanner(System.in);
        System.out.println("enter the number = ");
        n=ob.nextInt();
        for (x=n;n>0;n/=10)
        {
            s=(s*10)+n%10;}
            if (s==x)
                System.out.println("palindrome Number"+x);
            else
                System.out.println(" not palindrome Number"+x);
        
    } 
}